demo/src/app.js

```diff
@@ -92,36 +137,43 @@ const AppContent = () => {
     return (
         <IntlProvider locale={language} messages={messages[language]}>
             <ThemeProvider theme={lightTheme}>
-                <CssBaseline />
-                <TopBar
-                    appName="Demo"
-                    appColor="#808080"
-                    appLogo=<PowsyblLogo />
-                    onParametersClick={() => console.log('settings')}
-                    onLogoutClick={() => logout(dispatch, userManager.instance)}
-                    onLogoClick={() => console.log('logo')}
-                    user={user}
-                    appsAndUrls={apps}
-                />
-                {user !== null ? (
-                    <Box mt={20}>
-                        <Typography
-                            variant="h3"
-                            color="textPrimary"
-                            align="center"
-                        >
-                            Connected
-                        </Typography>
-                    </Box>
-                ) : (
-                    <AuthenticationRouter
-                        userManager={userManager}
-                        signInCallbackError={null}
-                        dispatch={dispatch}
-                        history={history}
-                        location={location}
+                <SnackbarProvider hideIconVariant={false}>
```

Comment:
I think we must add the maxSnack to define the maximum number of notifications to display